client/my-sites/feature-upsell/refund-asterisk.jsx

```diff
@@ -17,8 +17,8 @@ const RefundAsterisk = () => (
 		<div className="feature-upsell__refund-asterisk">
 			<p>Purchases made on WordPress.com can be cancelled and refunded during the refund period.</p>
 			<p>
-				For new domain registrations, you must cancel within 48 hours of purchase to receive a
-				refund. There are no refunds, pro-rated or otherwise, after 48 hours.
+				For new domain registrations, you must cancel within 96 hours of purchase to receive a
```

Comment:
Ummm, I wonder if these strings are not translated on purpose or by mistake?

Reply:
I noticed that too.  My understanding (could be incorrect though) is that these upsell pages are dead code and not currently being used.  They may have just been added as English-only to do a test on them?

Either way, I figured it was worth updating them for the 48=>96 hour refund window change :smiley:

Reply:
Ah, good point, maybe it's indeed dead 🤞 Definitely worth updating nevertheless, like you said, lets see what @adamziel says :)